core/geometry/src/polyface/Polyface.ts

```diff
@@ -30,94 +30,143 @@ import { PolyfaceData } from "./PolyfaceData";
 export abstract class Polyface extends GeometryQuery {
   /** String name for schema properties */
   public readonly geometryCategory = "polyface";
-
   /** Underlying polyface data. */
   public data: PolyfaceData;
+  /** Constructor */
   protected constructor(data: PolyfaceData) {
     super();
     this.data = data;
   }
-  /** create and return a visitor for this concrete polyface. */
+  /**
+   * Create and return a visitor for this concrete polyface.
+   * @param numWrap the number of vertices to replicate in visitor arrays.
+   */
   public abstract createVisitor(numWrap: number): PolyfaceVisitor;
   /** Flag indicating if the mesh display must assume both sides are visible. */
-  public get twoSided() { return this.data.twoSided; }
-  public set twoSided(value: boolean) { this.data.twoSided = value; }
-  /** Flag indicating if the mesh closure is unknown (0), open sheet (1), closed (2) */
-  public get expectedClosure(): number { return this.data.expectedClosure; }
-  public set expectedClosure(value: number) { this.data.expectedClosure = value; }
+  public get twoSided() {
+    return this.data.twoSided;
+  }
+  public set twoSided(value: boolean) {
+    this.data.twoSided = value;
+  }
+  /**
+   * Flag indicating if the mesh closure is unknown (0), open sheet (1), closed solid (2).
+   * * A boundary edge of a mesh is defined as an edge with only one connected facet.
+   * * Closed solid is a mesh with no boundary edge. Open sheet is a mesh that has boundary edge(s).
+   */
+  public get expectedClosure(): number {
+    return this.data.expectedClosure;
+  }
+  public set expectedClosure(value: number) {
+    this.data.expectedClosure = value;
+  }
   /**
-     * Check validity of indices into a data array.
-     * * It is valid to have  both indices and data undefined.
-     * * It is NOT valid for just one to be defined.
-     * * Index values at indices[indexPositionA <= i < indexPositionB] must be valid indices to the data array.
-     * @param indices array of indices.
-     * @param indexPositionA first index to test
-     * @param indexPositionB one past final index to test
-     * @param data data array
-     * @param dataLength length of data array
-     */
-  public static areIndicesValid(indices: number[] | undefined, indexPositionA: number, indexPositionB: number, data: any | undefined, dataLength: number): boolean {
+   * Check validity of indices into a data array.
+   * * It is valid to have both indices and data undefined.
+   * * It is NOT valid for just one to be defined.
+   * * Index values at indices[indexPositionA <= i < indexPositionB] must be valid indices to the data array.
+   * @param indices array of indices.
+   * @param indexPositionA first index to test.
+   * @param indexPositionB one past final index to test.
+   * @param data data array.
+   * @param dataLength length of data array.
+   */
```

Comment:
Failure of RotationalSweeps test might be related to changes in this function too. I will investigate.

Reply:
I will investigate the failure. Not sure if it relates to the change. If so, I can restore it.